### PR TITLE
Make buildbox-casd log level configurable

### PR DESCRIPTION
**Pull request by nanonyme**
Default is error

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -31,12 +31,15 @@ casd_metrics_args: >-
...
 casd_bind_path: ""
 casd_bind: "{{ 'unix:' ~ casd_bind_path if casd_bind_path else casd_bind_address ~ ':' ~ casd_port }}"
+# (debug, error, info, trace, warning) [optional, default = "error"]
+casd_loglevel: ""
 casd_quota_high: ""
 casd_reserved: ""
 casd_cache_mnt: "/srv"
 casd_certs_mnt: "/certs"
 casd_cmd: >-
   --verbose --bind {{ casd_bind }}
```
Comment (jjardon) [inline defaults/main.yml]: @nanonyme should we remove the "--verbose" from here?

Comment (nanonyme) [reply defaults/main.yml]: Definitely. Looks like I messed up with web UI. Unless someone beats me to it, I will probably fix in the morning.

Comment (nanonyme) [reply defaults/main.yml]: The point is log level needs to be configurable without touching this repo.

Comment (sdclarke) [reply defaults/main.yml]: I removed the hardcoded `--verbose` flag.

**Review by jjardon — APPROVED** [review]

**Pull request merged**